hack/testmanifestfmt/main.go

```diff
@@ -0,0 +1,121 @@
+package main
+
+import (
+	"context"
+	"flag"
+	"fmt"
+	"go/ast"
+	"go/format"
+	"go/parser"
+	"go/token"
+	"log"
+	"os"
+	"strings"
+
+	"github.com/hashicorp/go-version"
+	"github.com/hashicorp/hc-install/product"
+	"github.com/hashicorp/hc-install/releases"
+	"github.com/hashicorp/terraform-exec/tfexec"
+)
+
+func init() {
+	flag.Parse()
+}
+
+func main() {
+	ctx := context.Background()
+
+	log.Println("[INFO] installing terraform")
+
+	installer := &releases.ExactVersion{
+		Product: product.Terraform,
+		Version: version.Must(version.NewVersion("1.0.6")),
+	}
+
+	execPath, err := installer.Install(ctx)
+	if err != nil {
+		log.Fatalf("[ERROR] unable to intsall terraform: %s", err)
+	}
+
+	tmpdir := os.TempDir()
+	defer func() { _ = os.RemoveAll(tmpdir) }()
+
+	tf, err := tfexec.NewTerraform(tmpdir, execPath)
+	if err != nil {
+		log.Fatalf("[ERROR] unable to create terraform handle: %s", err)
+	}
+
+	log.Println("[INFO] installed terraform, formatting files")
+
+	for i := range flag.Args() {
+		path := flag.Args()[i]
+
+		log.Println("[INFO] formatting ", path)
+		if err = reformatFileAtPath(ctx, path, tf); err != nil {
+			log.Println("[ERROR]: unable to format ", path, ": ", err)
```

Comment:
`log.Formatf`?

Reply:
as discussed offline, ill dismiss the logging comments